src/main/java/mate/controller/driver/CreateDriverController.java

```diff
@@ -0,0 +1,33 @@
+package mate.controller.driver;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/drivers/add")
+public class CreateDriverController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/driver/createDriver.jsp")
+                .forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws IOException {
+        driverService.create(new Driver(req.getParameter("name"),
+                req.getParameter("license_number")));
+        resp.sendRedirect("web_practice_war/drivers");
```

Comment:
I think you can use here just "/drivers" without "web_practice_war"

Reply:
yeap, but it just for run and test, because I could be running and test only in this way :)

Reply:
I think you just configure tomcat incorrect. Try to go to Tomcat configuration, choose Deployment tab and set Application context just "/"

Reply:
And than you can remove web_practice_war from your sendRedirect